ModularBungalow/Secure_lockers.dm

```diff
@@ -12,19 +12,19 @@
 	new /obj/item/storage/backpack/satchel/cap(src)
 	new /obj/item/storage/backpack/duffelbag/captain(src)
 	new /obj/item/clothing/neck/cloak/cap(src)
-	new /obj/item/clothing/under/rank/captain/galaxy(src)
-	new /obj/item/clothing/under/rank/captain/skirt/galaxy(src)
-	new /obj/item/clothing/suit/armor/captain/galaxy(src)
-	new /obj/item/clothing/head/caphat/galaxy(src)
-	new /obj/item/clothing/under/rank/captain/parade(src)
+	new /obj/item/clothing/under/rank/command/captain/freelancer(src)
+	new /obj/item/clothing/under/rank/command/captain/freelancer/skirt(src)
+	new /obj/item/clothing/suit/armor/vest/capcarapace/freelancer(src)
```

Comment:
File this under armor/captain.
Almost all the captain armor is filed under here, except for the original (which will be moved)